Csn.Public.Dto/TestDrive/Infringement.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace Csn.Public.Dto
+{
+    public class Infringement
+    {
+        public Guid Identifier { get; set; }
+        public Guid? TestDriveIdentifier { get; set; }
```

Comment:
Does this need to be in the DTO? By inference it would have the parent identifier based on the object it was attached to?

Reply:
ah yes, test drive should contain a collection of infringements.